### DISCUSSION TITLE
MIDI2LR 5.4.0.0 and LrC 12.2 update problem and solution

### DISCUSSION THREAD

**Discussion by lfrancis** [rsjaffe/MIDI2LR#732] [Problems]
Recently updated LrC to 12.2 and suddenly Lightroom Classic stalled at launch, opening only if I forced a launch with default settings. Eventually removed and reinstalled MIDI2LR and now all is well. 

Unsure if this is actually related to the update or just a coincidental fail, but fwiw.

M1 MacBook Pro running Monterey 12.6.3

**Comment by robreyez** [comment]
Hey there.. Just wanted to get some input, if the plugin still works well with the new Lightroom as Im planning on picking up an X-Touch mini. Thanks., @lfrancis

**Comment by jkreibich** [comment]
I'm using MIDI2LR 5.4.0 w/ LrC 12.3 on an M1 MBP running Mac OS 12.4.   The X-Touch mini works perfectly.
